lexis360/parser.js

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env node
+
+'use strict';
+const Parser = require('../.lib/parser.js');
+
+/**
+ * Recognizes the accesses to the platform Lexis 360
+ * @param  {Object} parsedUrl an object representing the URL to analyze
+ *                            main attributes: pathname, query, hostname
+ * @param  {Object} ec        an object representing the EC whose URL is being analyzed
+ * @return {Object} the result
+ */
+module.exports = new Parser(function analyseEC(parsedUrl, ec) {
+  let result = {};
+  let path   = parsedUrl.pathname;
+  // uncomment this line if you need parameters
+  let param = parsedUrl.query || {};
+  let rawUrl = ec.url;
```

Comment:
This parser is a bit special: one of the query parameter contains # chars that prevent getting all the subsequent parameters to be parsed. That is why I'm relying on the (much more) cumbersome raw URL